tftrt/examples/cpp/image-classification/README.md

```diff
@@ -0,0 +1,54 @@
+# TF-TRT example for conversion in C++
+
+## Introduction
+
+This directory contains example code to demonstrate TF-TRT conversion using the C++ API.
+
+### Limitations
+- Freezing the model is not implemented correctly, the current example only works with frozen graphs.
+- Some of the intermediate steps are possibly inefficient. This can influence the conversion time. The inference runtime is not affected.
+
+### Acknowledgment
+
+The MNIST inference example is based on https://github.com/bmzhao/saved-model-example
+
+## How to run
+
+### Build TF
+```
+git clone --branch cpp_conversion_example https://github.com/tfeher/tensorrt.git
+git clone --branch trt_cpp_conversion_example https://github.com/tfeher/tensorflow.git tensorflow-source
+mkdir bazel-cache
+nvidia-docker run --rm -it --shm-size=1g --ulimit memlock=-1 --ulimit stack=67108864 -v $PWD:/data -w /data -v $PWD/bazel-cache:/root/.cache/bazel nvcr.io/nvidia/tensorflow:21.06-tf2-py3
```

Comment:
This nvidia-docker syntax is out of date? We should be using `docker run --gpus=all ... `?